lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
-#: messages.c:135
-msgid "G-code sliced for a different printer type. Please re-slice the model again. Print cancelled."
-msgstr ""
-
-# MSG_GCODE_NEWER_FIRMWARE_CONTINUE c=20 r=5
-#: util.cpp:381
-msgid "G-code sliced for a newer firmware. Continue?"
-msgstr ""
-
-# MSG_GCODE_NEWER_FIRMWARE_CANCELLED c=20 r=8
-#: util.cpp:387
-msgid "G-code sliced for a newer firmware. Please update the firmware. Print cancelled."
-msgstr ""
-
-# MSG_PREHEATING_TO_CUT c=20
-#: ultralcd.cpp:2309
-msgid "Preheating to cut"
-msgstr ""
-
-# MSG_PREHEATING_TO_EJECT c=20
-#: ultralcd.cpp:2306
-msgid "Preheating to eject"
-msgstr ""
-
-# MSG_NOZZLE_DIFFERS_CONTINUE c=20 r=5
-#: util.cpp:294
-msgid "Printer nozzle diameter differs from the G-code. Continue?"
-msgstr ""
-
-# MSG_NOZZLE_DIFFERS_CANCELLED c=20 r=9
-#: util.cpp:301
-msgid "Printer nozzle diameter differs from the G-code. Please check the value in settings. Print cancelled."
-msgstr ""
-
-# MSG_SELFTEST_FS_LEVEL c=20
-#: ultralcd.cpp:8195
-msgid "%s level expected"
-msgstr ""
-
-# MSG_RENAME c=18
-#: ultralcd.cpp:6579
-msgid "Rename"
-msgstr ""
-
-# MSG_SELECT c=18
-#: ultralcd.cpp:6572
-msgid "Select"
-msgstr ""
-
-# MSG_INFO_SENSORS c=18
-#: ultralcd.cpp:2041
-msgid "Sensor info"
-msgstr ""
-
-# MSG_SHEET c=10
-#: messages.c:65
-msgid "Sheet"
-msgstr ""
-
-# MSG_SOUND_BLIND c=7
-#: messages.c:146
-msgid "Assist"
-msgstr ""
-
-# MSG_STEEL_SHEET c=18
-#: messages.c:66
-msgid "Steel sheets"
-msgstr ""
-
-# MSG_Z_CORRECTION c=13
-#: ultralcd.cpp:5088
-msgid "Z-correct:"
-msgstr ""
-
-# MSG_Z_PROBE_NR c=14
-#: messages.c:149
-msgid "Z-probe nr."
-msgstr ""
-
-# MSG_PRINTER_IP c=18
-#: ultralcd.cpp:2028
-msgid "Printer IP Addr:"
-msgstr ""
-
+# Translation of Prusa-Firmware into Swedish.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Prusa-Firmware\n"
+"POT-Creation-Date: 2021-12-27 19:54+0100\n"
+"PO-Revision-Date: 2022-02-16 18:54+0100\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: sv\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 3.0.1\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+
+# MSG_IR_03_OR_OLDER c=18
+#: messages.c:164
+msgid " 0.3 or older"
+msgstr " 0.3 el \xe1ldre"
+
+# MSG_FS_V_03_OR_OLDER c=18
+#: Marlin_main.cpp:9884
+msgid "FS v0.3 or older"
+msgstr "FS v0.3 el \xe1ldre"
+
+# MSG_IR_04_OR_NEWER c=18
+#: messages.c:163
+msgid " 0.4 or newer"
+msgstr " 0.4 el nyare"
+
+# MSG_FS_V_04_OR_NEWER c=18
+#: Marlin_main.cpp:9883
+msgid "FS v0.4 or newer"
+msgstr "FS v0.4 el nyare"
+
+# MSG_IR_UNKNOWN c=18
+#: messages.c:165
+msgid "unknown state"
+msgstr "ok\xe1nt tillstand"
+
+# MSG_MEASURED_OFFSET c=20
+#: ultralcd.cpp:2913
+msgid "[0;0] point offset"
+msgstr "[0;0] punktf\xefrskjutn"
+
+# MSG_CRASH_DET_ONLY_IN_NORMAL c=20 r=4
+msgid "Crash detection can\\x0abe turned on only in\\x0aNormal mode"
+msgstr "Krockdetekt kan\\x0aendast aktiveras i\\x0anormalt l\xe1ge"
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.